browser/ai_chat/BUILD.gn

```diff
@@ -280,6 +312,7 @@ source_set("browser_tests") {
       "//brave/components/sidebar/browser",
       "//brave/components/text_recognition/common/buildflags",
       "//chrome/browser",
+      "//chrome/browser/actor",
```

Comment:
Why is this only a dep if `!is_android`? Should this be in a `!enable_brave_ai_chat_agent_profile` too?